_infra/helm/print-file/values.yaml

```diff
@@ -34,5 +34,7 @@ gcp:
   project: ras-rm-sandbox
   bucket:
     name: print-file
+  folder: 
```

Comment:
not sure this is particulaly clear config. here it seems to refer to a gcp folder which is essentially meaningless. i think it would be better for this to live under `gcp.bucket.prefix`

Reply:
done